utils_test.py

```diff
@@ -188,3 +191,48 @@ def test_get_thumbnail_icon_url_for_category(self):
         self.assertEqual(
             utils.get_thumbnail_icon_url_for_category('Nonexistent'),
             '/images/subjects/Lightbulb.svg')
+
+    def test_get_current_date_as_string(self):
```

Comment:
Somewhere in this test, you ought to check the actual value of the returned string...